VMBackup/main/freezesnapshotter.py

```diff
@@ -89,8 +89,10 @@ def update_snapshotinfoarray(self, blob_snapshot_info_array):
         snapshot_info_array = []
 
         self.logger.log('updating snapshot info array from blob snapshot info')
-        for blob_snapshot_info in blob_snapshot_info_array:
-            snapshot_info_array.append(Status.SnapshotInfoObj(blob_snapshot_info.isSuccessful, blob_snapshot_info.snapshotUri, blob_snapshot_info.errorMessage))
+        if blob_snapshot_info_array != None and blob_snapshot_info_array !=[]:
```

Comment:
Will below compatible with all python version. Test on python 2.6
blob_snapshot_info_array !=[]:

Reply:
i have tested. It worked fine.